images/router/haproxy/conf/haproxy-config.template

```diff
@@ -131,7 +131,7 @@ backend be_sni
 
 frontend fe_sni
   # terminate ssl on edge
-  bind 127.0.0.1:{{env "ROUTER_SERVICE_SNI_PORT" "10444"}} ssl no-sslv3 {{ if (len .DefaultCertificate) gt 0 }}crt {{.DefaultCertificate}}{{ else }}crt /var/lib/haproxy/conf/default_pub_keys.pem{{ end }} crt {{ $workingDir }}/certs accept-proxy
+  bind 127.0.0.1:{{env "ROUTER_SERVICE_SNI_PORT" "10444"}} ssl no-sslv3 crt /tmp/def-router.pem crt {{ $workingDir }}/certs accept-proxy
```

Comment:
Looks like this references a path that is only sometimes set up in the run_haproxy script